api/src/lib/interface.ts

```diff
@@ -670,15 +670,16 @@ export interface OrcidAffiliationDate {
  * - Qualifications
  */
 export interface MappedOrcidAffiliation {
-    id: number;
+    id: number | string;
     affiliationType:
         | 'membership'
         | 'service'
         | 'invited-position'
         | 'distinction'
         | 'employment'
         | 'education'
-        | 'qualification';
+        | 'qualification'
+        | 'misc.';
```

Comment:
what exactly is this? Do we need it?

Reply:
I added it because of the AC: “Affiliation Type” is populated with “Misc.”
So any new-style affiliation created by this script will have this type.

Reply:
hmm, ok. This interface is also used on the frontend to map over and display affiliations. Might worth checking if it won't break the UI

Reply:
It looked OK when I viewed one. The value was displayed and capitalised i.e. "Misc."